cmd/node/flags.go

```diff
@@ -372,6 +372,12 @@ var (
 		Value: "./config/p2pKey.pem",
 	}
 
+	// snapshotsEnabled is used to enable snapshots, if it is not set it defaults to true, it will be set to false if it is set specifically
```

Comment:
is the default true enforced somewhere? I do not see a IsSet check before the assignment

Reply:
yes, it is BoolTFlag, it default to true if it is not set explicitly as false

Reply:
:) ok